wordpress-mobile/WordPress-iOS
# Preliminary implementation of Latest Post Summary Chart View

### Description 
Fixes #11061, which calls for the creation of a chart for _Latest Post Summary_, backed by stubbed data.

Notes:

- The first four commits represent administrivia; I alphabetized a couple of things before applying my changes.
- The fabricated data is based off of the REST API endpoint documentation. Once the Core Data work is complete, the protocol-based approach should make it relatively straightforward to transition to a real data source via #11067.
- The appearance of the bars in this Chart differ slightly from Zeplin. It appears that it is _possible_ to round the corners of the bar tops, but it is not something you get for free. I intend to investigate & implement this next.

To test:

- Checkout the branch & confirm that existing tests pass.
- Navigate to _Stats_ for a selection of Sites. While this Stats view presents fabricated data, it does so in a manner congruent with existing behavior. That is to say, the conditions that currently evince either the _Create Post_ or _Share Post_ states are still valid.

Update release notes:


### Evidence

**iPhone SE**
![11061a](https://user-images.githubusercontent.com/221062/53205114-36378500-35e2-11e9-8c16-8d39d795e891.gif)

**iPad (Portrait)**
<img width="535" alt="ipadportrait" src="https://user-images.githubusercontent.com/221062/53205159-59623480-35e2-11e9-8e5c-f476f07df6f0.png">
